### PR TITLE
Iaedev 61822 New naicscodes field has been added

### PR DESCRIPTION
**Pull request by kalyaninaljala**

### CODE REVIEW

File: _apidocs/get-opportunities-public-api.md [hunk 1]
```diff
@@ -89,6 +89,7 @@ setAside	|Set Aside Description	|String | v1<br> v2
 setAsideCode	|Set Aside Code	|String | v1<br> v2
 reponseDeadLine	|Response Deadline Date	|String | v1<br> v2
 naicsCode	|NAICS Code. This code is maximum of 6 digits	|String | v1<br> v2
+naicsCodes	|NAICS Codes. This array contains multiple codes.This code is maximum of 6 digits  |String[] | v1<br> v2
```
Comment (TaehoKimGSA) [inline _apidocs/get-opportunities-public-api.md:92]: There are multiple examples and you also need to update those response example as well.

**Review by TaehoKimGSA — CHANGES_REQUESTED** [review]

File: _apidocs/fh-fouo-api.md [hunk 2]
```diff
@@ -199,6 +199,9 @@ Note: Response for one record is provided as an example <br>
       "typeOfSetAside": "7",
       "responseDeadLine": null,
       "naicsCode": "236220",
+	  "naicsCodes": [
```
Comment (TaehoKimGSA) [inline _apidocs/fh-fouo-api.md:202]: I am not sure why you want to modify FH document. Can you verify?

File: _apidocs/get-opportunities-public-api.md [hunk 3]
```diff
@@ -216,6 +217,9 @@ Note: Response for one record is provided as an example <br>
             "typeOfSetAside": "7",
             "responseDeadLine": null,
             "naicsCode": "236220",
+			"naicsCodes": [
```
Comment (TaehoKimGSA) [inline _apidocs/get-opportunities-public-api.md:220]: There is indent issue and "naicsCodes" field is not lined up well. Can you fix it?

File: _apidocs/get-opportunities-public-api.md [hunk 4]
```diff
@@ -371,6 +375,9 @@ Note: Response for one record is provided as an example <br>
       "typeOfSetAside": "7",
       "responseDeadLine": null,
       "naicsCode": "236220",
+	  "naicsCodes": [
```
Comment (TaehoKimGSA) [inline _apidocs/get-opportunities-public-api.md:378]: There is indent issue and "naicsCodes" field is not lined up well. Can you fix it?

File: _apidocs/get-opportunities-public-api.md [hunk 5]
```diff
@@ -462,6 +469,9 @@ Note: Response for one record is provided as an example <br>
             "typeOfSetAside": null,
             "responseDeadLine": null,
             "naicsCode": "236220",
+			"naicsCodes":[
```
Comment (TaehoKimGSA) [inline _apidocs/get-opportunities-public-api.md:472]: There is indent issue and "naicsCodes" field is not lined up well. Can you fix it?

File: _apidocs/get-opportunities-public-api.md [hunk 6]
```diff
@@ -709,6 +719,9 @@ Note: Response for one record is provided as an example <br>
       "typeOfSetAside": null,
       "responseDeadLine": "2018-05-07T20:00:00-05:00",
       "naicsCode": "336350",
+	  "naicsCodes": [
```
Comment (TaehoKimGSA) [inline _apidocs/get-opportunities-public-api.md:722]: There is indent issue and "naicsCodes" field is not lined up well. Can you fix it?

File: _apidocs/get-opportunities-public-api.md [hunk 7]
```diff
@@ -778,6 +791,7 @@ Note: Response for one record is provided as an example <br>
             "typeOfSetAside": null,
             "responseDeadLine": null,
             "naicsCode": null,
+			"naicsCodes": [],
```
Comment (TaehoKimGSA) [inline _apidocs/get-opportunities-public-api.md:794]: There is indent issue and "naicsCodes" field is not lined up well. Can you fix it?

**Review by TaehoKimGSA — CHANGES_REQUESTED** [review]

File: _apidocs/get-opportunities-public-api.md (hunk 3 above)
Comment (kalyaninaljala) [reply _apidocs/get-opportunities-public-api.md]: Resolved

File: _apidocs/get-opportunities-public-api.md (hunk 4 above)
Comment (kalyaninaljala) [reply _apidocs/get-opportunities-public-api.md]: resolved

File: _apidocs/get-opportunities-public-api.md (hunk 7 above)
Comment (kalyaninaljala) [reply _apidocs/get-opportunities-public-api.md]: resolved

**Review by TaehoKimGSA — APPROVED** [review]

**Pull request merged**